### PR TITLE
Created the root partition reallocation script

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: docs/cloning.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 #### On the Destination node (blank host)
 
-1. Make sure that the Destination node has adequate space and also make sure that the root partition has all the storage space. You may utilize this [script] (https://gist.githubusercontent.com/sthirugn/cdc34006ae280c344a15a474f7e35918/raw/d6382b8ffabcd9b2a5ab07150201388abfb7e01f/reallocate.sh) if needed.
+1. Make sure that the Destination node has adequate space and also make sure that the root partition has all the storage space. You may utilize this [script] (../helpers/reallocate.sh) if needed.
```
Comment (pgagne) [inline docs/cloning.md]: @sthirugn perhaps this should be "You may utilize the script in ```<script-path``` instead of a link to the file?

Comment (sthirugn) [inline docs/cloning.md]: do you mean like this?

https://raw.githubusercontent.com/satellite-clone/helpers/reallocate.sh

Comment (pgagne) [inline docs/cloning.md]: My suggestion is to change this line to:

1. Make sure that the Destination node has adequate space and also make sure that the root partition has all the storage space. You may utilize the ```helpers/reallocate.sh``` script in the playbook for this.

**Comment by pgagne** [comment]
ACK

**Pull request merged**